apps/wpcom-block-editor/src/calypso/iframe-bridge-server.js

```diff
@@ -92,10 +95,24 @@ function transmitDraftId( calypsoPort ) {
  * @param {MessagePort} calypsoPort Port used for communication with parent frame.
  */
 function handlePostTrash( calypsoPort ) {
+	//avoid navigation to wp-admin page
 	$( '#editor' ).on( 'click', '.editor-post-trash', e => {
```

Comment:
Do we really need to cancel this? According to the code, Gutenberg only dispatches the `trashPost` action ([#](https://github.com/WordPress/gutenberg/blob/1ca616a456a9d9c5ab4698bf67589a99e1802fff/packages/editor/src/components/post-trash/index.js#L14)) which we are already customizing, so it seems there is nothing else to prevent here.

Not a blocker, but it would be great if we can confirm we don't need this anymore and remove it before merging if that's the case.

Reply:
I don't recall if earlier versions had a direct link to wp-admin or not, we can maybe remove otherwise.

Reply:
Removing the jQuery action appears to do the trick ✅